languages/js/src/Oso.test.ts

```diff
@@ -13,17 +13,15 @@ describe('Equality function used for unification', () => {
     const o = new Oso();
     o.registerConstant('undefined', undefined);
     o.registerConstant('null', null);
-    expect(() => o.loadStr('?= undefined = null;')).not.toThrow(
-      InlineQueryFailedError
-    );
+    o.loadStr('f() if undefined = null;');
+    expect(Array.from(o.query('f()'))).toStrictEqual([map()]);
```

Comment:
OK, but why not just `o.query("undefined = null")`?

Reply:
just wasn't the first thing I thought of. updated